services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -121,6 +121,21 @@ def storage_endpoint(self) -> str:
         return f"{self.STORAGE_HOST}:{self.STORAGE_PORT}"
 
 
+class CatalogSettings(BaseCustomSettings):
```

Comment:
settings of common services (e.g. postgres) should be in ``setting_library``. IN any case, i think @GitHK refactored this in his PR

Reply:
yes.